### ISSUE TITLE
Nexson specification of tree inference method

### ISSUE THREAD

**Issue by josephwb** [OpenTreeOfLife/opentree#552]
The curator calls the tree inference method ```"Tree type"```. [Nexsons](https://github.com/OpenTreeOfLife/phylesystem-api/wiki/NexSON#table-i-predicate-vocabulary) call this ```"ot:curatedType"```. 

The nexson property name should be changed to be more informative.

Posting this here since it involves multiple tools: the curator, UI, and validation.

**Comment by jar398** [comment]
Since it conveys the inference method, maybe the property should be called
ot:inferenceMethod, and the curator app should present it as "Inference
method". Just sayin'.

**Comment by mtholder** [comment]
Just chasing down the history: the earliest reference I found to this was in this tread: https://github.com/OpenTreeOfLife/phylografter/issues/86
the notes ( https://docs.google.com/document/d/1tVXWaq3kqctlCh3qonkXxCphLeEJi0_k-3kSx949sQE/edit#heading=h.wk263bxa5r6t ) from the Oct 9, 2013 which @pmidford  alludes to don't specifically mention that tag, but do refer to a free-form text field describing the tree type.

**Comment by jimallman** [comment]
> Since it conveys the inference method, maybe the property should be called ot:inferenceMethod, and the curator app should present it as "Inference method"

I think the problem is that it _may_ contain other information, at least in the studies from phylografter. In OpenTreeOfLife/phylografter#86, @pmidford says it has historically held other information:

> From Oct 9 call
> ot:curatedType (tree) - phylografter's type column - frequently contains inference method, but also other free text.

Maybe we should check for other values in the current phylesystem studies.

Hm, looks like the curation UI is actually inconsistent in its labeling. Here are a tree's properties when we're editing:

![screen shot 2015-01-19 at 9 34 46 am](https://cloud.githubusercontent.com/assets/446375/5802002/c01339c2-9fbe-11e4-9408-69abb2975143.png)

...and here's the same tree when we're just viewing the study (note the label for this field!):

![screen shot 2015-01-19 at 9 35 04 am](https://cloud.githubusercontent.com/assets/446375/5802004/c8a2eb3c-9fbe-11e4-88be-4875d647a35a.png)


I'll make a separate issue for this...

**Comment by pmidford** [comment]
Perhaps we should add ot:inferenceMethod, the curator UI seems pretty 
clear about what should be there.  It would be nice to enforce a 
controlled vocabulary, but that is probably premature. Leave ot:type 
until someone can triage whatever else is in the free text for the 
phylografter studies.

**Comment by mtholder** [comment]
agreed that we need a controlled vocabulary.  I ran

    python scripts/phylesystem/ot_phylesystem_list_propery_values.py --report-ids  --tree '^ot:curatedType'

(on a recently updated version of a peyotl script). And retrieved the list of values for this field (and their associated trees) as shown in: 
   https://docs.google.com/spreadsheets/d/1ki4mcSCNnl4z1uxUhCgENxCBFc104ua0hsn7UYU3jGw/edit#gid=100428982

it is a bit of a mess, as free text fields tend to be...

**Comment by josephwb** [comment]
From that list, it seems that ```^ot:curatedType``` often contains information that should appear for the tree ```label```.
From the curator:
![curator](https://cloud.githubusercontent.com/assets/808499/5803578/5de67746-9fcd-11e4-8750-505be1c1826f.png)
In the nexson:
![nexson](https://cloud.githubusercontent.com/assets/808499/5803582/6e7c4df6-9fcd-11e4-9651-e68d1b898d05.png)

**Comment by kcranston** [comment]
I agree with Joseph about label. Having label be free text is fine (as a means to distinguish trees within studies) but that we want a controlled vocabulary on inference type. I would also like to start capturing information about tree type (gene tree, supermatrix, supertree, species tree), noting that 'tree type' might not be the best name for that field.

**Comment by jimallman** [comment]
> I would also like to start capturing information about tree type (gene tree, supermatrix, supertree, species tree), noting that 'tree type' might not be the best name for that field.

Are these values mutually exclusive? 

Is **Tree type** an optional choice, or should it be required?

**Comment by jimallman** [comment]
See related discussion in #555.
